conf/nd96isr_v5.conf

```diff
@@ -0,0 +1,107 @@
+ * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/run"
+ * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/run/lock"
+ * || check_fs_mount_rw -t "pstore" -s "pstore" -f "/sys/fs/pstore"
+ * || check_fs_mount_rw -t "efivarfs" -s "efivarfs" -f "/sys/firmware/efi/efivars"
+ * || check_fs_mount_rw -t "bpf" -s "bpf" -f "/sys/fs/bpf"
+ * || check_fs_mount_rw -t "hugetlbfs" -s "hugetlbfs" -f "/dev/hugepages"
+ * || check_fs_mount_rw -t "mqueue" -s "mqueue" -f "/dev/mqueue"
+ * || check_fs_mount_rw -t "debugfs" -s "debugfs" -f "/sys/kernel/debug"
+ * || check_fs_mount_rw -t "tracefs" -s "tracefs" -f "/sys/kernel/tracing"
+ * || check_fs_mount_rw -t "fusectl" -s "fusectl" -f "/sys/fs/fuse/connections"
+ * || check_fs_mount_rw -t "configfs" -s "configfs" -f "/sys/kernel/config"
+ * || check_fs_mount_ro -t "ramfs" -s "none" -f "/run/credentials/systemd-sysusers.service"
+ * || check_fs_mount_ro -t "squashfs" -s "/dev/loop0" -f "/snap/core20/1852"
+ * || check_fs_mount_ro -t "squashfs" -s "/dev/loop1" -f "/snap/lxd/24322"
+ * || check_fs_mount_rw -t "vfat" -s "/dev/sda15" -f "/boot/efi"
+ * || check_fs_mount_rw -t "tracefs" -s "tracefs" -f "/sys/kernel/debug/tracing"
+ * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/run/snapd/ns"
+ * || check_fs_mount_rw -t "nsfs" -s "nsfs" -f "/run/snapd/ns/lxd.mnt"
+ * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/run/user/1000"
+ * || check_fs_used / 90%
+ * || check_fs_used /dev 90%
+ * || check_fs_used /boot/efi 90%
+ * || check_fs_used /snap/snapd/18933 100%
+ * || check_fs_iused / 100%
+ * || check_fs_iused /dev 98%
+ * || check_fs_iused /snap/core20/1852 12k
+ * || check_fs_iused /snap/core20/1879 12k
+
+
+#######################################################################
+###
+### Hardware checks
+###
+ * || check_hw_cpuinfo 2 96 96
+ * || check_hw_physmem 1915071MB 1915071MB 5%
+ * || check_hw_swap 0kB 0kB 3%
+ * || check_hw_ib 400 mlx5_ib0:1
+ * || check_hw_ib 400 mlx5_ib1:1
+ * || check_hw_ib 400 mlx5_ib2:1
+ * || check_hw_ib 400 mlx5_ib3:1
+ * || check_hw_ib 400 mlx5_ib4:1
+ * || check_hw_ib 400 mlx5_ib5:1
+ * || check_hw_ib 400 mlx5_ib6:1
+ * || check_hw_ib 400 mlx5_ib7:1
+ * || check_hw_eth lo
+ * || check_hw_eth eth0
+ * || check_hw_eth ib0
+ * || check_hw_eth ib1
+ * || check_hw_eth ib2
+ * || check_hw_eth ib3
+ * || check_hw_eth ib4
+ * || check_hw_eth ib5
+ * || check_hw_eth ib6
+ * || check_hw_eth ib7
+ * || check_hw_eth docker0
+
+
+#######################################################################
+###
+### Process checks
+###
+ * || check_ps_loadavg 192
+ * || check_ps_service -r rpcbind
+ * || check_ps_service -S -u root auditd
+
+
+#######################################################################
+####
+#### GPU checks
+####
+ * || check_nvsmi_healthmon
+ * || check_app_gpu_clocks
+ * || check_cuda_bw 52
+ * || check_gpu_ecc 20000000 10000
+ * || check_gpu_clock_throttling
+ * || check_nccl_allreduce 350.0 1 /opt/microsoft/ndv5-topo.xml
```

Comment:
What's the message size that we are checking here? Depending on that, we may have to bump this up.

Reply:
1G message. ANy suggestions?

Reply:
I think we can use larger message size (8G or 16G), and have a larger threshold limit.

Reply:
we will address